functions/src/index.ts

```diff
@@ -165,27 +165,26 @@ export const addInvitedUserToBoardOwners = https.onCall(
         }
         const boardOwners = boardSnapshot.data()?.owners
 
-        await getFirestore()
-            .collection('settings/' + data.boardId + '/invites')
-            .where('reciever', '==', context.auth?.token.email)
-            .limit(1)
-            .get()
-            .then((inviteDoc) => inviteDoc.docs[0].id)
-            .catch(() => {
-                throw new https.HttpsError(
-                    'permission-denied',
-                    'Requested user does not have an invite for this board',
-                )
-            })
+        try {
+            await getFirestore()
+                .collection('settings')
+                .doc(data.boardId)
+                .collection('invites')
+                .where('receiver', '==', context.auth.token.email)
+                .limit(1)
+                .get()
```

Comment:
Eg skjønner det sånn at dette vil kræsje p.g.a. Security Rules, men eg tenker kanskje at det er fint å ha ein sjekk i kode òg? Kanskje mest for sikkerhets skyld, om me skulle brekke security rules?

Då tenker eg at du kan kaste samme feil dersom resultatet av spørringa er tom liste, f.eks.

Reply:
Høres fornuftig ut, legger det inn:)